Formula/couchdb.rb

```diff
@@ -60,15 +67,63 @@ def install
     system "make"
     system "make", "install"
 
+    install_geocouch if build.with? "geocouch"
+
     # Use our plist instead to avoid faffing with a new system user.
     (prefix/"Library/LaunchDaemons/org.apache.couchdb.plist").delete
     (lib/"couchdb/bin/couchjs").chmod 0755
-    (var/"lib/couchdb").mkpath
-    (var/"log/couchdb").mkpath
-    (var/"run/couchdb").mkpath
+  end
+
+  def geocouch_share
+    share/"couchdb-geocouch"
+  end
+
+  def install_geocouch
```

Comment:
Normally I am but this is a strange case so your arguments are persuasive. Feel free to leave as-is.